src/WorldIDIdentityManagerImplV1.sol

```diff
@@ -181,24 +159,11 @@ contract WorldIDIdentityManagerImplV1 is WorldIDImpl, IWorldID {
     /// @param latestRoot The actual latest root at the time of the transaction.
     error NotLatestRoot(uint256 providedRoot, uint256 latestRoot);
 
-    /// @notice Thrown when attempting to enable the bridge when it is already enabled.
-    error StateBridgeAlreadyEnabled();
-
-    /// @notice Thrown when attempting to disable the bridge when it is already disabled.
-    error StateBridgeAlreadyDisabled();
-
-    /// @notice Thrown when attempting to set the state bridge address to the zero address.
-    error InvalidStateBridgeAddress();
-
```

Comment:
Note that the optimizer should remove unused code, so I don't think this would have any affect on the code size. However, if these are no longer used, it should be fine to remove.